pkg/debug/diff.go

```diff
@@ -0,0 +1,111 @@
+package debug
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/v31/github"
+	"github.com/kylelemons/godebug/diff"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// GenerateDiff attempts to pull a dependency list from a previous job (job, jobID) and generate a diff against a provided string
+func GenerateDiff(baseURL, jobName, jobID, phase, dependencies string) (string, error) {
+	parsedJobID, err := strconv.Atoi(jobID)
```

Comment:
Let's just make JobID an integer in config -- that should eliminate our need to do this.

Otherwise, this LGTM.

Reply:
Good call

Reply:
That was a little bit of trouble lol